src/main/java/org/alfresco/utility/data/TestData.java

```diff
@@ -357,5 +377,41 @@ public DSLJmx withJMX()
     {
         return new DSLJmx(jmxBuilder.getJmxClient());
     }
-    
+
```

Comment:
I believe the REST API `GET /sites/{siteId}/containers/documentLibrary` can be used here to retrieve the nodeRef of the documentLibrary node.

Reply:
Yes, changed..thanks